addons/account/models/account_move.py

```diff
@@ -180,7 +180,7 @@ def _get_default_invoice_incoterm(self):
     # ==== Business fields ====
     fiscal_position_id = fields.Many2one('account.fiscal.position', string='Fiscal Position', readonly=True,
         states={'draft': [('readonly', False)]},
-        domain="[('company_id', '=', company_id)]",
+        domain="[('company_id', '=', company_id)]",ondelete="restrict",
```

Comment:
```suggestion
        domain="[('company_id', '=', company_id)]", ondelete="restrict",
```
It's pure nitpicking, but could add a little space ? The code is more pleasing to read with some spaces :)

Reply:
Ya my mistake , actually i squashed real commit message. 
I will update with proper commit now.